receiver/prometheusreceiver/internal/otlp_metricfamily_test.go

```diff
@@ -161,8 +161,8 @@ func TestMetricGroupData_toSummaryUnitTest(t *testing.T) {
 						{Name: "a", Value: "A"}, {Name: "quantile", Value: "0.0"}, {Name: "b", Value: "B"},
 					},
 					scrapes: []*scrape{
-						{at: 10, value: 10, metric: "histogram_count"},
-						{at: 10, value: 12, metric: "histogram_sum"},
+						{at: 10, value: 10, metric: "summary_count"},
```

Comment:
Why the metric name changes here? Was histogram incorrect?

Reply:
@codeboten the method name is `TestMetricGroupData_toSummaryUnitTest(t *testing.T)` and it is testing `summary` metric, but the `metric` string was mentioned as `histogram`, so I corrected it to `summary`.